.readthedocs.yml

```
@@ -1,10 +1,14 @@
 # .readthedocs.yml
 
+version: 2
+
 build:
   image: latest
+  tools:
+    python: 3.9
```

This should be a string according to the docs.
https://docs.readthedocs.io/en/stable/config-file/v2.html#build-tools-python
```suggestion
    python: "3.9"
```